src/components/EditImageModal/EditImageModal.test.jsx

```diff
@@ -0,0 +1,770 @@
+import React from 'react';
+import { CheckBox, Modal, InputText } from '@edx/paragon';
+
+import EditImageModal from './index';
+import messages from './displayMessages';
+import rewriteStaticLinks from '../../utils/rewriteStaticLinks';
+import WrappedMessage from '../../utils/i18n/formattedMessageWrapper';
+import { mountWithIntl } from '../../utils/i18n/enzymeHelper';
+
+let wrapper = mountWithIntl(<EditImageModal />);
+let modalBody;
+let formContainer;
+let insertImageButton;
+let imageSourceURLInput;
+let imageDescriptionContainer;
+let imageDescriptionInput;
+let imageDescriptionInputCheckBox;
+let imageDimensionsContainer;
+let imageDimensionsWidthInput;
+let imageDimensionsHeightInput;
+let imageDimensionsInputCheckBox;
+
+const learnMoreLink = 'http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/accessibility/best_practices_course_content_dev.html#use-best-practices-for-describing-images';
+
+const sampleText = 'edx.jpg';
+
+const sampleImgData = {
+  naturalWidth: 100,
+  naturalHeight: 200,
+};
+sampleImgData.aspectRatio = sampleImgData.naturalWidth / sampleImgData.naturalHeight;
+
+const updateConsants = () => {
+  wrapper.update();
+  modalBody = wrapper.find('.modal-body');
+  formContainer = modalBody.find('div.col form');
+
+  insertImageButton = wrapper.find('.modal-footer button').first();
+
+  imageSourceURLInput = formContainer.find(InputText).filterWhere(input => input.prop('name') === 'imageSourceURL');
+
+  imageDescriptionContainer = formContainer.find('fieldset').at(0);
+  imageDescriptionInput = formContainer.find('fieldset').find(InputText).filterWhere(input => input.prop('name') === 'imageDescription');
+  imageDescriptionInputCheckBox = imageDescriptionContainer.find(CheckBox);
+
+  imageDimensionsContainer = formContainer.find('fieldset').at(1);
+  imageDimensionsWidthInput = formContainer.find('fieldset .form-row .col').find(InputText).filterWhere(input => input.prop('name') === 'imageWidth');
+  imageDimensionsHeightInput = formContainer.find('fieldset .form-row .col').find(InputText).filterWhere(input => input.prop('name') === 'imageHeight');
+  imageDimensionsInputCheckBox = imageDimensionsContainer.find(CheckBox);
+};
+
+// use to remount the EditImageModal at the start of a test for a clean slate
```

Comment:
If I remember correctly, Michael was having some issues with getting code in a `beforeEach` to run for all of the tests in this file. I think it was because `beforeEach`s are local to the `describe` block that contains it.